edgecontext.thrift

```diff
@@ -16,6 +16,12 @@ e.g. en, en_US
 */
 typedef string LocaleCode
 
+/** One of supported locale codes forced to the BCP-47 format 
+(e.g. de-DE, pt-BR, etc.)
+
+*/
+typedef string NegotiatedLocaleCode
```

Comment:
having type name to be the same as field name is really redundant and not useful. the name of the type should reflect the type itself, not the field using it. we should name the type something like `BCP47LocaleCode`.